### ISSUE TITLE
Find MAC address of my Dinamica PLUS 

### ISSUE THREAD

**Issue by AirmakZ** [Arbuzov/home_assistant_delonghi_primadonna#4]
HI guys.
I'm trying to connect to my Dinamica Plus coffee machine using this integration.
First step is.... "Find the device MAC address using BLE scanner or smartphone"... here I'm...

Im trying to scan ble devices using "bluetoothctl" from linux. It list a lot of devices, some of them with manufacturer description as "Flower Care", "MI_SCALE, etc and several ones without description. just MAC address...

My question is how are you finding the proper one?
I tried to communicate via app in order to see some extra traffic... but frankly speaking I could not detect something interesting...

I will agree some tips.
Thanks in advanced!

**Comment by Arbuzov** [comment]
Well, I used this two guys in windows:
https://www.microsoft.com/en-us/p/ble-scanner/9nblggh0j7m0
https://apps.microsoft.com/store/detail/bluetooth-le-explorer/9N0ZTKF1QD98
Or BLE scanner for android:
https://play.google.com/store/apps/details?id=com.macdom.ble.blescanner&hl=ru&gl=US
To identify the hostname I used native android app.
![image](https://user-images.githubusercontent.com/1662280/179370744-6830a4f2-72c7-4f68-aa03-6121a5775909.png)
And then by the host name I got the MAC:
![image](https://user-images.githubusercontent.com/1662280/179370724-e07c8651-5728-4163-ba75-90df5bb89456.png)

That was a hard exercise, but that worth it.

**Comment by AirmakZ** [comment]
Hi Sergey!
Unfortunately I don't have neither Windows nor Android. And BLS Scanner in iOS is not showing MAC address.

BUT ! Even from linux I don't know I cannot see Dxxxxxxxxx device description I searched a similar range of Mac address taking yours as reference... and... I could find mine !! 00:A0:50:50:7C:62

I performed a 'power on' command already and it works perfectly!
Time to continue playing now.

Thanks for your support.
